packages/types/src/types/models-extra/mapOverlay.ts

```diff
@@ -70,6 +70,16 @@ type MeasureConfig = {
    */
   hideFromLegend?: boolean;
 
+  /**
+   * @description Whether to include this series in the popup/tooltip
+   */
+  hideFromPopup?: boolean;
```

Comment:
This is actually already supported in the app code but the types weren't correct so the setting couldn't be used in the viz builder.